tests/test_config.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+import pytest
+import srsly
+from hugie.config import app
+from hugie.models import InferenceEndpointConfig
+from typer.testing import CliRunner
+
+from common import INCOMPLETE_JSON
+
+runner = CliRunner()
+
+
+@pytest.fixture(scope="function")
+def data_path(tmp_path):
+    data_path = os.path.join(tmp_path, "data.json")
+    data = {
+        "accountId": None,
+        "compute": {
+            "accelerator": "cpu",
+            "instanceSize": "small",
+            "instanceType": "c6i",
+            "scaling": {"maxReplica": 1, "minReplica": 1},
+        },
+        "model": {
+            "framework": "custom",
+            "image": {"huggingface": {}},
+            "repository": "t5-small",
+            "revision": "main",
+            "task": "translation",
+        },
+        "name": "development",
+        "provider": {"region": "us-east-1", "vendor": "aws"},
+        "type": "protected",
+    }
+    srsly.write_json(data_path, data)
+
+    return data_path
+
+
+def test_inference_endpoint_config_serialization(data_path):
+    config = InferenceEndpointConfig.from_json(data_path)
+    config = dict(config)
+    assert isinstance(config, dict)
```

Comment:
maybe better to test that some data are still there instead of lost

Reply:
will add

Reply:
Added in 667d83a